From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Chainguard Engineers <noreply@chainguard.dev>
Date: Mon, 24 Feb 2026 00:00:00 +0000
Subject: [PATCH] Add envoy patches

Patch istio WORKSPACE to add patches to the vendored envoy dependency.
---
 WORKSPACE | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/WORKSPACE b/WORKSPACE
index 40acf942..e4933ff3 100644
--- a/WORKSPACE
+++ b/WORKSPACE
@@ -35,6 +35,11 @@ ENVOY_REPO = "envoy"
 # persist the option in `user.bazelrc`.
 http_archive(
     name = "envoy",
+    patch_args = ["-p1"],
+    patches = [
+        "//:0001-envoy-remove-sysroot.patch",
+    ],
     sha256 = ENVOY_SHA256,
     strip_prefix = ENVOY_REPO + "-" + ENVOY_SHA,
     url = "https://github.com/" + ENVOY_ORG + "/" + ENVOY_REPO + "/archive/" + ENVOY_SHA + ".tar.gz",
